src/sinks/aws_cloudwatch_logs/mod.rs

```diff
@@ -502,7 +502,7 @@ async fn healthcheck(
         return Ok(());
     }
 
-    let group_name = String::from_utf8_lossy(&config.group_name.get_ref()[..]).into_owned();
+    let group_name = config.group_name.get_ref().to_owned();
```

Comment:
There may be an opportunity to do this in other places in this sink.

Reply:
I changed `group` and `stream` type to String.